lib/crowbar/init/helpers.rb

```diff
@@ -208,9 +208,8 @@ def api_constraint(*versions)
           break if versions_requested.nil?
           return true if versions_requested[:major].to_i == major &&
               versions_requested[:minor].to_i <= minor
```

Comment:
the `return` within the block will cause `LocalJumpError` if the conditions apply